xformers/sparse/_csr_ops.py

```diff
@@ -126,7 +126,7 @@ def forward(
         ctx, b, row_indices, values, row_offsets, column_indices, m, _transp_info
     ):
         out = torch.ops.xformers.spmm_sputnik(
-            b, row_indices, values, row_offsets, column_indices, m
+            b.contiguous(), row_indices, values, row_offsets, column_indices, m
```

Comment:
nit: if you do

```python
b = b.contiguous()
```
then you don't need to perform the `.contiguous()` call in the backward, as it will be stored directly as a contiguous tensor. This will make things slightly faster